packages/@ama-sdk/schematics/schematics/typescript/core/openapi-codegen-typescript/src/main/resources/typescriptFetch/spec/api-mock.mustache

```diff
@@ -1,6 +1,6 @@
 {{#apiInfo}}
 import { type ApiClient, isApiClient } from '@ama-sdk/core';
-import { ApiFetchClient, type BaseApiFetchClientConstructor } from '@ama-sdk/client-fetch';
+import { ApiFetchClient, type BaseApiFetchClientConstructor } from '@ama-sdk/core';
```

Comment:
Should we keep both lines separated?

Reply:
I would say yes, it is not leading to any issue and will reduce the diff change when migrating to `client-fetch` :)